src/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/AbstractLspLogger.cs

```diff
@@ -11,9 +11,13 @@ namespace Microsoft.CommonLanguageServerProtocol.Framework;
 
 internal abstract class AbstractLspLogger : ILspLogger
 {
-    public abstract void LogDebug(string message, params object[] @params);
+    [Obsolete($"Use {nameof(CreateContext)} instead.")]
```

Comment:
I think it may be possible to just delete these (assuming the clasp source package stuff is all working right)

Reply:
Going to go for the delete.